docs/user/forms.md

```diff
@@ -64,7 +63,7 @@ To flesh out this overview:
   - `/V`: Field Value
   - `/DV` : Default Field Value (used when resetting a form for example)
 * In order to streamline readability, _Field_ Objects and _Widget_ Objects can be fused housing all properties.
-* Fields can be organised hierarchically, id est one field can be placed under another. In such instances, the `/Parent` will have an IndirectObject providing Bottom-Up links and `/Kids` is an array carrying IndirectObjects for Top-Down navigation; _Widget_ Objects are still required for visual rendering. To call upon them, use the *fully qualified field name* (where all the individual names of the parent objects are separated by `.`)
```

Comment:
'id est' is the latin expression. I propose 'i.e.' instead of 'that is' 😉

Reply:
Abbreviations tend to be discouraged in written communication, thus "id est" is completely fine.